controllers/authenticationController.js

```diff
@@ -18,4 +19,30 @@ const registerUser = async (req, res) => {
     res.status(500).send(err)
   }
 }
-module.exports = { registerUser }
\ No newline at end of file
+const loginUser = async (req, res) => {
+  const { username, password } = req.body;
+  try {
+    if (!username || !password) {
+      res.status(404).send({ message: "Username or password can not be empty" })
+    } else {
+      const dbUser = await userModel.findOne({ where: { username } })
+      if (dbUser) {
+        const passwordMatch = await bcrypt.compare(password, dbUser.password);
+        if (passwordMatch) {
+          const userAuthenticationToken = jwt.sign({
+            data: `${username} + ${password}`
```

Comment:
you need to store more data in JWT token, other wise, how are going to know, if the token belongs to an admin user or a normal user @ashok-choudhary-0